packages/generator/src/utils/fetch-latest-version-for.ts

```diff
@@ -19,7 +19,7 @@ export const fetchLatestVersionsFor = async <T extends Record<string, string>>(
       async ([dep, version]): Promise<[string, string]> => {
         let skipFetch = false
 
-        if (!version.match(/\d.x/)) skipFetch = true
+        if (!version.match(/\d.x/) && !dep.match(/@prisma\/\w+/)) skipFetch = true
```

Comment:
could this check be described by a short comment, saying why `@prisma` stuff is handled explicitly?